Formal power series and polynomials in commutative rings (#1541)
UniMath/SymmetryBookFormalization: src/elementary-number-theory/binary-sum-decompositions-natural-numbers.lagda.md

      ( is-section-map-inv-equiv-permute-components-triple-with-sum-pr2)
      ( is-retraction-map-inv-equiv-permute-components-triple-with-sum-pr2)
```

> ### The binary sum decomposition of 0 is contractible

Reviewer: Or
```suggestion
### There is a unique binary sum decomposition of 0
```
